Userland/Libraries/LibGfx/Painter.cpp

```diff
@@ -1229,10 +1229,10 @@ ALWAYS_INLINE static void do_draw_scaled_bitmap(Gfx::Bitmap& target, IntRect con
                 auto bottom_left = get_pixel(source, scaled_x0, scaled_y1);
                 auto bottom_right = get_pixel(source, scaled_x1, scaled_y1);
 
-                auto top = top_left.interpolate(top_right, x_ratio);
-                auto bottom = bottom_left.interpolate(bottom_right, x_ratio);
+                auto top = top_left.mixed_with(top_right, x_ratio);
+                auto bottom = bottom_left.mixed_with(bottom_right, x_ratio);
```

Comment:
`interpolate` is always inlined while `mixed_with` is out of line, did you do some measurements to see the performance impact? We might want to inline `mixed_with` as well if we see some significant decrease in performance.

Reply:
`mixed_with()` probably should be inline, the only reason it's out of line is because that made it easier to work on when I was implementing it (since touching `Color.h` is a full rebuild :sweat_smile:)